docs/prod-fluo-setup.md

```diff
@@ -65,23 +46,20 @@ When you have a distribution tarball built for your environment, follow these st
 to install Fluo.
 
 First, choose a directory with plenty of space and untar the distribution:
-```
-tar -xvzf fluo-1.0.0-beta-1-SNAPSHOT-bin.tar.gz
-```
+
+    tar -xvzf fluo-1.0.0-beta-1-SNAPSHOT-bin.tar.gz
```

Comment:
I think we should.  I found `SNAPSHOT` in a few spots in the docs.  Fixed them all in fbdf401647f0f